skeletons/phpcs.xml

```diff
@@ -3,4 +3,82 @@
     <description>Coding standards for SilverStripe based project</description>
 
     <rule ref="vendor/jdolba/silverstripe-coding-standards/definitions/php/phpcs-ss4.xml" />
+
+    <!-- Don't sniff third party libraries -->
+    <exclude-pattern>./vendor/*</exclude-pattern>
+    <exclude-pattern>*/thirdparty/*</exclude-pattern>
```

Comment:
these can be defined in composer json script definition

I would like to keep definition files only to use include/exclude rules as much as possible